include/mbgl/math/wrap.hpp

```diff
@@ -9,6 +9,11 @@ namespace util {
 // arithmetic.
 template <typename T>
 T wrap(T value, T min, T max) {
+    if (value >= min && value < max) {
+        return value;
+    } else if (value == max) {
+        return min;
+    }
```

Comment:
If we are going with an if ladder, might as well write
```c++
if (value < min) {
  return min + std::fmod(value - min, d) + d;
} else if (value < max) {
  return value;
} else if (value == max) {
  return min;
} else {
  return min + std::fmod(value - min, d);
}
```
That is IMO easier to read than `std::fmod((std::fmod((value - min), d) + d), d) + min`.

But anyway, LGTM!

Reply:
Fair point @anderco :+1: will update the code accordingly.